### PR TITLE
Root keys named :wallet/... moved as nested under :wallet root key

### LINKED ISSUE #18724: Wallet - cleanapp db to only have one root key
The wallet db in status-im.subs.root has too many root keys for the wallet for the approach that the team has decided to take. Reduce it down to a single root key as suggested.

### PR DESCRIPTION
**Pull request by vkjr**
fixes #18724

### Summary

Root keys in app-db named `:wallet/...` moved as nested under `:wallet` root key.

status: ready

### CODE REVIEW

File: src/status_im/contexts/wallet/events.cljs [hunk 1]
```diff
@@ -303,62 +304,67 @@
...
+        (assoc-in [:wallet :search-address :valid-ens-or-address?]
+                  (boolean result)))}))
 
 (rf/reg-event-fx :wallet/fetch-address-suggestions
  (fn [{:keys [db]} [_address]]
-   {:db (assoc db
-               :wallet/local-suggestions     nil
-               :wallet/valid-ens-or-address? false)}))
+   {:db (-> db
+            (assoc-in [:wallet :search-address :local-suggestions] nil)
```
Comment (J-Son89) [inline src/status_im/contexts/wallet/events.cljs]: maybe put it in `:ui`, i,e `(assoc-in [:wallet :ui :search-address :local-suggestions] nil)`
cc @ulisesmac

Comment (vkjr) [reply src/status_im/contexts/wallet/events.cljs]: yeah, good idea!

File: src/status_im/subs/wallet/wallet.cljs [hunk 2]
```diff
@@ -267,3 +267,18 @@
  :wallet/transactions
  :<- [:wallet]
  :-> :transactions)
+
+(rf/reg-sub
```
Comment (J-Son89) [inline src/status_im/subs/wallet/wallet.cljs:271]: some tests would be great too! 🙏

**Review by J-Son89 — APPROVED** [review]

**Review by ulisesmac — APPROVED** [review]
Very needed PR! thanks @vkjr  :+1:

**Comment by ulisesmac** [comment]
@vkjr  I recently merged some refactored subs and events using an approach @J-Son89 suggested:

https://github.com/status-im/status-mobile/pull/19043#discussion_r1508016493

For the code in this PR I'm not completely sure, but probably the subscriptions could also follow it.

**Comment by pavloburykh** [comment]
@vkjr thanks for the pr. E2E results are okay. In case manual qa is not required PR is ready for merge.

**Pull request merged**